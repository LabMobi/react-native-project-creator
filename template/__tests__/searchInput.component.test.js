import React from 'react';
import renderer from 'react-test-renderer';
import SearchInput from '../src/components/common/searchInput';

describe('searchInput component', () => {
  it('renders searchInput correctly', () => {
    const tree = renderer
      .create(<SearchInput handleChangeDebounce={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
