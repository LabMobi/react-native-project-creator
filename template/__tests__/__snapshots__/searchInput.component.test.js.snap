// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchInput component renders searchInput correctly 1`] = `
<TextInput
  onChangeText={[Function]}
  placeholder="Search..."
  placeholderTextColor="rgb(0, 122, 255)"
  style={
    Array [
      Object {
        "alignSelf": "center",
        "borderColor": "silver",
        "borderRadius": 10,
        "borderWidth": 1,
        "height": 40,
        "marginVertical": 10,
        "paddingLeft": 10,
        "width": "90%",
      },
      Object {
        "backgroundColor": undefined,
        "color": "rgb(28, 28, 30)",
      },
    ]
  }
/>
`;
